eth/downloader/downloader.go

```diff
@@ -110,6 +113,11 @@ func New(mux *event.TypeMux, hasBlock hashCheckFn, getBlock getBlockFn) *Downloa
 		hashCh:    make(chan hashPack, 1),
 		blockCh:   make(chan blockPack, 1),
 	}
+	// Inject all the known bad hashes
+	downloader.banned = make(map[common.Hash]struct{})
+	for hash, _ := range core.BadHashes {
+		downloader.banned[hash] = struct{}{}
+	}
```

Comment:
I made the copy to be able to add to it later if we decide. Should I just assume we won't add to it for now?